src/molecules/Stepper/Stepper.tsx

```diff
@@ -22,51 +30,47 @@ const Container = styled.div<ContainerProps>`
 `;
 
 export interface StepperProps {
-  current: number;
-  setCurrent: (value: number) => void;
-  steps: string[];
   onlyShowCurrentStepLabel?: boolean;
   maxWidth?: string;
 }
 
 export const Stepper: FC<StepperProps> = ({
-  current,
-  setCurrent,
-  steps,
   onlyShowCurrentStepLabel = false,
   maxWidth,
 }) => {
+  const { steps, currentStep } = useStepper();
   const children = useMemo((): ReactElement[] => {
     const all: ReactElement[] = [];
     steps.forEach((step, index) => {
       all.push(
         <Step
           key={`step-${index}`}
           index={index}
-          currentIndex={current}
-          setCurrentIndex={setCurrent}
           onlyShowCurrentStepLabel={onlyShowCurrentStepLabel}
         >
-          {step}
+          {step.label}
         </Step>
       );
 
       if (index !== steps.length - 1) {
         all.push(
-          <StepLine key={`step-line-${index}`} done={current > index} />
+          <StepLine key={`step-line-${index}`} done={currentStep > index} />
         );
       }
     });
     return all;
-  }, [current, onlyShowCurrentStepLabel, setCurrent, steps]);
+  }, [currentStep, onlyShowCurrentStepLabel, steps]);
 
   return (
-    <Container
-      $stepAmount={steps.length}
-      $maxWidth={maxWidth}
-      data-testid="stepper-container"
-    >
-      {children}
-    </Container>
+    <Wrapper>
+      <Container
+        $stepAmount={steps.length}
+        $maxWidth={maxWidth}
+        data-testid="stepper-container"
+      >
+        {children}
```

Comment:
The children name confused me at first, thought it came from a prop from the user, where they could pass some custom content or something, so this could be renamed to something else i think 🤔 Like content or steps